src/utils/migrate.ts

```diff
@@ -49,8 +49,11 @@ export async function readMigrations (migrationPath?: string) {
               return reject(err)
             }
 
-            const [up, down] = data.split(/^--\s+?down\b/im)
+            const [up, down] = data
+              .split(/^--\s+?down\b/im)
+              .map(value => value.replace(/^-{3,}$/gim, '').trim())
 
+            delete migration.filename
```

Comment:
Why remove it?

Reply:
For the types consistency.

Do you want to save all parameters from migration that may be add in the future?

So maybe we will make another type in Migration class for the readMigrations function and in the migrate function we will just require needed arguments but not the exact object as MigrationData.

What do you think?

Reply:
That sounds appropriate